.github/workflows/stack.yaml

```diff
@@ -0,0 +1,114 @@
+name: random-CI
+
+on:
+  push:
+    branches: [ '*' ]
+  pull_request:
+    branches: [ master ]
+
+defaults: { run: { shell: bash } }
+
+jobs:
+  build:
+    name: CI
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
```

Comment:
Are you sure we do not want `fail-fast: true`?

Reply:
Yes, Ia m sure. I wanna see all failures, instead of the first one. It is always easy to cancel the whole build if quick termination is needed

Reply:
I'm concerned that in future GitHub Actions might become less generous for OSS, and we'll have to minimise build time. All right, let's worry about it later.

Reply:
> I'm concerned that in future GitHub Actions might become less generous for OSS

We'll go back to Travis-CI then :laughing: 

Let's not worry about it, Microsoft is rich, so let's just hope they won't get stingy.